BP/common/casus_belli_types/00_vassalization.txt

```diff
@@ -0,0 +1,773 @@
+﻿vassalization_cb = {
```

Comment:
Do CBs in the same group have to be in the same file, or could the tributary CB be separate?

Reply:
They do not have to be the same; probably had some issue with it earlier and forgot to split it.